ledger/sandbox-common/src/main/scala/platform/sandbox/config/SandboxConfig.scala

```diff
@@ -176,7 +176,7 @@ object SandboxConfig {
       sqlStartMode = Some(DefaultSqlStartupMode),
       enableCompression = false,
       enableSelfServiceErrorCodes = true,
-      userManagementConfig = UserManagementConfig.default,
+      userManagementConfig = UserManagementConfig.default(true),
```

Comment:
Can you please remind me why the default is different between kvutils and sandboxes?

Reply:
Talked offline